subsystems/provisioning/definitions.go

```diff
@@ -298,9 +300,11 @@ func ConfigFromJSON(defaultConf Config, jsonBytes []byte) (*Config, error) {
 	}
 
 	if conf.DeviceRebootAfterOfflineMinutes != 0 &&
-		conf.DeviceRebootAfterOfflineMinutes < conf.OfflineTimeout ||
-		conf.DeviceRebootAfterOfflineMinutes < conf.UserTimeout {
-		return &conf, errw.Errorf("device_reboot_after_offline_minutes cannot be less than offline_timeout or user_timeout")
+		(conf.DeviceRebootAfterOfflineMinutes < conf.OfflineTimeout || conf.DeviceRebootAfterOfflineMinutes < conf.UserTimeout) {
```

Comment:
It took me WAY too long to figure out this was the issue. One pair of missing parenthesis! So easy to miss!